README.md

```diff
@@ -629,6 +629,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Pippo](http://www.pippo.ro/) - Small, highly modularized Sinatra-like framework.
 * [Play](https://www.playframework.com/) - Uses convention over configuration, hot code reloading and display of errors in the browser.
 * [PrimeFaces](http://primefaces.org/) - JSF framework which has a free and a commercial version with support. Provides several frontend components.
+* [Rapidoid](http://www.rapidoid.org/) - An Open source (Apache license), Simple, Secure and Extremely fast Web Framework for Java.
```

Comment:
Sure, have modified the description based on above feedback to 
"A Simple, Secure and Extremely fast Web Framework consisting of embedded HTTP server, GUI components and Dependency Injection."